core/package.go

```diff
@@ -9,22 +9,20 @@ package core
 
 import (
 	"fmt"
-	"gopkg.in/yaml.v2"
 	"io/ioutil"
 	"os"
-	"time"
+
+	"gopkg.in/yaml.v2"
 )
 
 type Package struct {
 	Name    string
 	Title   string
-	Author  string            "author,omitempty"
-	Version string            "version,omitempty"
-	Require []string          "require,omitempty"
-	Binary  map[string]string "binary,omitempty"
-	// ModTime is currently used only for setting the modification time of local
-	// packages. It is ignored by the YAML parser.
-	ModTime time.Time "-"
+	Author  string            `yaml:"author,omitempty"`
+	Version string            `yaml:"version,omitempty"`
+	Require []string          `yaml:"require,omitempty"`
+	Binary  map[string]string `yaml:"binary,omitempty"`
+	Created string            `yaml:"created,omitempty"`
```

Comment:
So you actually do need `string` for this datetime? Once we are ready to implement the package update functionality - how will this work?

Reply:
Turns out that we're better off implementing our own Marshaller for Date field. If we use the built-in, then we're no longer able to work with packages that are either missing the Created field or have it in unexpected format. But now, with a custom field, this is all being cared of.